client/src/components/Form.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
```

Comment:
You could use `styled-components/macro` which would display your components name as class name in the browser. It's useful to quickly navigate in your developer tools.
```suggestion
import styled from 'styled-components/macro';
```